Project/Assets/Prairie/Framework/Script/Annotation/ParseAnnotation.cs

```diff
@@ -3,8 +3,8 @@
 
 public static class ParseAnnotation {
 
-    private static readonly string IMAGE_TEXT_FULL = "<\\s*img\\s*=\\s*\"?.*?\"?\\s*>";
-    private static readonly string IMAGE_TEXT_SPLIT = "(<\\s*img\\s*=\\s*\"?)|(\"?\\s*>)";
+    private static readonly string IMAGE_TEXT_FULL = "<\\s*img\\s*src\\s*=\\s*\"?.*?\"?\\s*>";
+    private static readonly string IMAGE_TEXT_SPLIT = "(<\\s*img\\s*src\\s*=\\s*\"?)|(\"?\\s*>)";
 
```

Comment:
> doesn't need to be resolved in this PR

I disagreed with Jeff on this change. There's nothing more intuitive about the exact syntax of the HTML image tag, other than perhaps familiarity with folks with web dev experience (which our project doesn't target).

What about the Markdown way? [Or the way iA Writer does image embedding](https://ia.net/writer/support/general/content-blocks/), by just detecting a file path on its own line?
```
/images/Structure.jpg
```